### DISCUSSION TITLE
Can't surpass 18xx token context with 13B 8K?

### DISCUSSION THREAD

**Discussion by viperwasp** [oobabooga/text-generation-webui#2890] [Q&A]
I downloaded Wizard Vicuna 13B Super Hot 8K GPTQ. I have a RTX 4080 16GB. VRAM use automatically goes to 97-99% upon loading. 

I followed these steps. 

How to easily download and use this model in text-generation-webui with ExLlama

Please make sure you're using the latest version of text-generation-webui

    Click the Model tab.
    Under Download custom model or LoRA, enter TheBloke/Wizard-Vicuna-13B-Uncensored-SuperHOT-8K-GPTQ.
    Click Download.
    The model will start downloading. Once it's finished it will say "Done"
    Untick Autoload the model
    In the top left, click the refresh icon next to Model.
    In the Model dropdown, choose the model you just downloaded: Wizard-Vicuna-13B-Uncensored-SuperHOT-8K-GPTQ
    To use the increased context, set the Loader to ExLlama, set max_seq_len to 8192 or 4096, and set compress_pos_emb to 4 for 8192 context, or to 2 for 4096 context.
    Now click Save Settings followed by Reload
    The model will automatically load, and is now ready for use!
    Once you're ready, click the Text Generation tab and enter a prompt to get started!

Kept throwing questions at the model in instruct mode. I could not get context to raise past 1800 or so? Do I have to chance settings on another page too? Or is this all I need to do?

**Comment by viperwasp** [comment]
I almost wonder if 16GB is not enough for 8K 13B? But I would think it would still go above 2K if I set 8K in the settings? I might test it at 4K later.

**Reply by valden80** [reply]
I have 3060 12Gb. On linux with fully unloaded GUI from the card, I can use slightly more then 4K context with 13B 4bit superhot models loaded through exllama. I think, 16Gb for 8K context may be enough, but only so-so. I can be wrong, but 2k context take 8-9Gb, and 4k - ~11.8Gb on my card.

**Reply by viperwasp** [reply]
Thanks I will try to figure out what I am doing wrong etc. I have not played around with it sense. I might try 4K next just to see if that works first as a troubleshooting step. You provided helpful information. I should easily be able to use 4K.  If I can't there is an issue somewhere.
